This kernel patch from the squashfs-3.1r2 release does not compile on my
armeb-linux-gcc 4.1.1 when optimizing for size (-Os). This works around
that problem by using optimization flag -O2 instead for these two files.

Leon Woestenberg <leonw@mailcan.

--- linux-2.6.17/fs/squashfs/Makefile	2006-10-10 09:57:03.000000000 +0200
+++ linux-2.6.17-squashfs_arm/fs/squashfs/Makefile	2006-10-10 09:57:53.000000000 +0200
@@ -5,3 +5,6 @@
 obj-$(CONFIG_SQUASHFS) += squashfs.o
 squashfs-y += inode.o
 squashfs-y += squashfs2_0.o
+
+CFLAGS_squashfs2_0.o = "-O2"
+CFLAGS_inode.o = "-O2"
gcc 3.4.4 for sh4
